packages/d/doublecmd/package.yml

```diff
@@ -37,13 +37,13 @@ setup      : |
     mv doublecmd.png doublecmd-qt.png
     mv doublecmd.sh doublecmd-qt.sh
 build      : |
-    cd doublecmd-${version}
+    # cd doublecmd-${version}
```

Comment:
Since this is commented out, can it be removed entirely?

Reply:
It can, but I didn't remove it on purpose. I was thinking that, if in future updates it will not be necessary, then I will remove it.